presto-native-execution/presto_cpp/presto_protocol/tests/Base64Test.cpp

```diff
@@ -82,6 +82,47 @@ TEST_F(Base64Test, singleTinyint) {
   ASSERT_EQ(1, intVector->valueAt(0));
 }
 
+TEST_F(Base64Test, simpleLongDecimal) {
+  // Unscaled value = 0
+  const std::string data0 =
+      "DAAAAElOVDEyOF9BUlJBWQEAAAAAAAAAAAAAAAAAAAAAAAAAAA==";
```

Comment:
These string values seem cryptic. Can we directly cast the unscaled values?

Reply:
I guess there is metadata involved. Can we construct the payload explicitly?

Reply:
@markjin1990 Did you push changes to address this comment? It would be easier to verify what exact value been passed in the encoded string if you construct the payload.

Reply:
@markjin1990 how are these payloads even built now? Can you add a comment?

Reply:
> @markjin1990 how are these payloads even built now? Can you add a comment?

Just added. I basically synthesized some records of the corresponding decimal values in the table and print the value of the 2nd parameter of `base64Encoded` of `readBlock` to the screen in online running service of Prestissimo when reading these synthesized values. Hope that is clear. 

Sorry that I haven't found an easy approach automatically synthesizing these values within the unit test and nor the other unit tests have demonstrated their approaches getting these values in the same test file.